ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -96,48 +108,49 @@ class CommandBarButton: UIButton {
 
     private let isPersistSelection: Bool
 
-    private var selectedTintColor: UIColor {
-        guard let window = window else {
-            return UIColor(light: Colors.communicationBlue,
-                           dark: .black)
-        }
+    private var normalTintColor: UIColor {
+        return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground1])
+    }
 
-        return UIColor(light: Colors.primary(for: window),
-                       dark: .black)
+    private var selectedTintColor: UIColor {
+        return UIColor(light: UIColor(colorValue: fluentTheme.aliasTokens.colors[.brandForeground4].light),
+                       dark: UIColor(colorValue: fluentTheme.aliasTokens.colors[.foreground1].dark!))
```

Comment:
why do we need "!" for dark colors?

Reply:
The dark ColorValue parameter is optional so I need to unwrap it

Reply:
> The dark ColorValue parameter is optional so it needs to be unwrapped

Reply:
We should never force unwrap, as it will crash if something goes wrong. You'll need to do a safe unwrapping elsewhere and feed the value into the UIColor constructor.

Reply:
It'll now directly pull the right global token if the dark alias color hasn't been set. Please let me know if this approach works